### ISSUE TITLE
Support loading Definitions from multiple modules in the pyproject.toml file

### ISSUE THREAD

**Issue by bendnorman** [dagster-io/dagster#11439]
### What's the use case?

 I’m able to load multiple definitions from modules using the -m argument:
```
dagit -m module_1 -m module_2
```
It would be nice to be able to specify multiple modules in the pyproject.toml file so multiple modules can be loaded by just using the dagit command.

### Ideas of implementation

I figure the end result could look like this:

```
[tool.dagster]
module_name = ["module_1", "module_2]
```

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by bendnorman** [comment]
I'm happy to take a stab at this if a maintainer can point me in the right direction. Thanks!

**Comment by bendnorman** [comment]
This was closed by #22327!
